src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -900,12 +901,47 @@ DropDefaultColumnDefinition(Oid relationId, char *columnName)
 	char *qualifiedRelationName = generate_qualified_relation_name(relationId);
 	const char *quotedColumnName = quote_identifier(columnName);
 
-	StringInfo sequenceDropCommand = makeStringInfo();
-	appendStringInfo(sequenceDropCommand,
+	StringInfo dropDefaultColumnDefCommand = makeStringInfo();
+	appendStringInfo(dropDefaultColumnDefCommand,
+					 ColumnIsGeneratedStored(relationId, columnName) ?
+					 "ALTER TABLE %s ALTER COLUMN %s DROP EXPRESSION" :
 					 "ALTER TABLE %s ALTER COLUMN %s DROP DEFAULT",
 					 qualifiedRelationName, quotedColumnName);
 
-	ExecuteAndLogUtilityCommand(sequenceDropCommand->data);
+	ExecuteAndLogUtilityCommand(dropDefaultColumnDefCommand->data);
+}
+
+
+static bool
+ColumnIsGeneratedStored(Oid relationId, char *targetColumnName)
```

Comment:
should do something like in `DistributionColumnUsesGeneratedStoredColumn` to fix pg11 builds

Reply:
71ad1bf